### ISSUE TITLE
User expandable harmonization

### ISSUE THREAD

**Issue by navtej** [certtools/intelmq#1315]
Should we have user defined harmonization fields as an extension to existing fields?
We can have a file `user-harmonization.conf` to define custom harmonization types and load those values along with  standard values. This needs bit more thinking on storage and sharing of data aspects. Thoughts?

**Comment by wagner-certat** [comment]
Yeah, in the next versions I want work on making intelmq more pluggable, see also #972 and others.
So then bot's could also add their own required fields to the harmonization.

Idea:
instead of having a single `harmonization.conf` use a `harmonization.conf.d`, where both other components/python packages and users can add their own files.

**Comment by aaronkaplan** [comment]
covered in the intelmq 3.0 architecture
